mqtt:
  client:
    protocol: 4      # MQTTv311
  connection:
    host: localhost
    port: 1883
    keepalive: 60
  private_path: device/001
serializer: json:dumps
deserializer: json:loads
wrm: ${HOME}/autoware_web_runtime_manager_local/autoware_app/mqtt_setting/config.json
bridge:
  # ndt_pose
  - factory: mqtt_bridge.bridge:RosToMqttBridge
    msg_type: geometry_msgs.msg:PoseStamped
    topic_from: /current_pose
    topic_to: ndt_pose
  # ndt_pose_for_map
  - factory: mqtt_bridge.bridge:RosToMqttBridge
    msg_type: geometry_msgs.msg:PoseStamped
    topic_from: /current_pose
    topic_to: map_pose
  # points_raw  
  - factory: mqtt_bridge.bridge:RosToMqttBridge
    msg_type: sensor_msgs.msg:PointCloud2
    topic_from: /downsampled_points_raw
    topic_to: points_raw
  # tf  
  - factory: mqtt_bridge.bridge:RosToMqttBridge
    msg_type: tf2_msgs.msg:TFMessage
    topic_from: /tf
    topic_to: tf
  # vector_map  
  - factory: mqtt_bridge.bridge:RosToMqttBridge
    msg_type: visualization_msgs.msg:MarkerArray
    topic_from: /vector_map
    topic_to: vector_map
  # lane_waypoints_array
  - factory: mqtt_bridge.bridge:RosToMqttBridge
    msg_type: autoware_msgs.msg:LaneArray
    topic_from: /lane_waypoints_array
    topic_to: lane_waypoints_array
  # downsampled_next_target_mark
  - factory: mqtt_bridge.bridge:RosToMqttBridge
    msg_type: visualization_msgs.msg:Marker
    topic_from: /downsampled_next_target_mark
    topic_to: downsampled_next_target_mark
  # downsampled_trajectory_circle_mark	    
  - factory: mqtt_bridge.bridge:RosToMqttBridge
    msg_type: visualization_msgs.msg:Marker
    topic_from: /downsampled_trajectory_circle_mark
    topic_to: downsampled_trajectory_circle_mark
  # clock
  - factory: mqtt_bridge.bridge:RosToMqttBridge
    msg_type: rosgraph_msgs.msg:Clock
    topic_from: /Clock
    topic_to: clock
  # initial_pose
  - factory: mqtt_bridge.bridge:MqttToRosBridge
    msg_type: geometry_msgs.msg:PoseWithCovarianceStamped
    topic_from: initialpose
    topic_to: /initialpose

